### PR TITLE
Blob reagent fixes, features, and updates

### PR DESCRIPTION
**Pull request by ghost**
Changes the messages of a few reagents, and changes the colors to be more clear on others.

Nerfs the following reagents:
- Lung-Destroying Toxin now does less oxygen damage and no toxin damage
- Explosive Gelatin now only has a 25% chance to explode, otherwise doing brute and burn damage
- Skin Melter no longer does burn damage, just ignites
- Morphine blob now injects considerably less morphine (about 30% of what it used to)

Adds the following reagents:
- Weakness Toxin - Dark reddish. Deals 10 brute damage and 5-20 stamina damage. Small chance to weaken on hit.
- Decomposing Anima - Purplish blue. Deals 10 brute damage and spawns an animated blob.  Animated blobs are slimes the same color as the blob with 1 health that die after 5 seconds. These slimes are weak and do 5-7 damage.
- Liquid Entropy: Dark gray. Has ten varying random effects upon hit. Expect everything from a spontaneous corgi to your voice being changed.

Subject to changes, please give feedback.

### CODE REVIEW

**Comment by WJohn** [comment]
Wait but I thought morphine already was a sedative type blob? Wouldn't this one be strictly better, considering it has no upside for the crew?

Also we already have a toxins and radioactive blob, they already can knock you out and poison you, this is a bit redundant.

Splitting anima sounds pretty neat, but how much damage do the animated blob pieces do?

**Comment by Xhuis** [comment]
In the next commit, the slimes will do 3-5 damage.

**Comment by Man-Shroom** [comment]
They should really, REALLY do 1 damage. Also, they should not block shots or walking, the same way you can walk over a mouse or shoot over it.

**Comment by Xhuis** [comment]
I did some testing and them doing 1 damage was insanely underpowered. They would literally die before dealing three damage. They currently do 5-7 damage - they are essentially glass cannons.

**Comment by Anonus** [comment]
Why did you change all the on hit messages now they're not even consistent between each other

**Comment by Xhuis** [comment]
Because they were kekworthy.

File: code/modules/reagents/Chemistry-Reagents/Blob-Reagents.dm [hunk 1]
```diff
@@ -166,10 +170,65 @@ datum/reagent/blob/spacedrugs
...
+datum/reagent/blob/blob_animated_slime/reaction_mob(var/mob/living/M as mob, var/method=TOUCH, var/volume, var/show_message = 1)
+	if(method == TOUCH)
+		if(show_message)
+			M << "<span class = 'userdanger'>Part of the blob breaks off and slams into you!</span>"
+		new /mob/living/simple_animal/slime/blob(get_turf(M))
+		M.apply_damage(10, BRUTE)
+
+/mob/living/simple_animal/slime/blob
+	name = "blob slime"
+	desc = "You should never see this."
```
Comment (Cheridan) [inline code/modules/reagents/Chemistry-Reagents/Blob-Reagents.dm]: Except you will. Gold slime reactions.

**Comment by Xhuis** [comment]
I'm fairly sure slimes are blacklisted from gold slime reactions, and being a subtype of slime I would imagine this is blacklisted too. Correct me if I'm wrong

File: code/modules/reagents/Chemistry-Reagents/Blob-Reagents.dm [hunk 2]
```diff
@@ -166,10 +170,65 @@ datum/reagent/blob/spacedrugs
...
+	health = 1
+	maxHealth = 1
+	rabid = 1
+	ventcrawler = 0
+
+/mob/living/simple_animal/slime/blob/New()
+	..()
+	spawn(0)
+		name = "animated blob"
+		desc = "A disturbing mass of moving liquid."
```
Comment (Anonus) [inline code/modules/reagents/Chemistry-Reagents/Blob-Reagents.dm]: No, you will NEVER see the above come on Cheridan

**Comment by Xhuis** [comment]
Conflicts fixed up

**Comment by Xhuis** [comment]
Added the Liquid Entropy reagent, which is basically the blob version of the staff of chaos.

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 3]
```diff
@@ -403,6 +403,8 @@ emp_act
 		var/damage = rand(5, 25)
 		if(M.is_adult)
 			damage = rand(10, 35)
+		if(istype(M, /mob/living/simple_animal/slime/blob))
```
Comment (Cheridan) [inline code/modules/mob/living/carbon/human/human_defense.dm]: snowflake code. add a damage var to slimes or something.

**Comment by Xhuis** [comment]
I can tell there's no real interest in merging this, so I'll scrap it for now.

**Pull request closed without merging**